slackclient/_client.py

```diff
@@ -36,7 +36,7 @@ def rtm_read(self):
             raise SlackNotConnected
 
     def rtm_send_message(self, channel, message):
-        return self.server.channels.find(channel).send_message(message)
+        return self.server.channels.find(channel.lstrip('#')).send_message(message)
```

Comment:
This is actually unnecessary because the comparison on channel happens here and with/without # is valid:

https://github.com/slackhq/python-slackclient/blob/master/slackclient/_channel.py#L9